test/OpenTelemetry.Exporter.Zipkin.Tests/ZipkinExporterTests.cs

```diff
@@ -554,7 +554,10 @@ public void UpdatesServiceNameFromIConfiguration()
 
         if (status.HasValue)
         {
+#pragma warning disable
             activity.SetStatus(status.Value);
+#pragma warning enable
+            activity.SetStatus((ActivityStatusCode)status.Value.StatusCode);
```

Comment:
@ysolomchenko - I think this assert is causing the issue in tests. I would just remove it for now. As a follow up, we can add additional test for validating the precedence like https://github.com/open-telemetry/opentelemetry-dotnet/pull/5781#discussion_r1717157416